prime-dotnet-webapi/Services/CHESApiService.cs

```diff
@@ -0,0 +1,173 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+
+namespace Prime.Services
+{
+    public class CHESApiService : BaseService, ICHESApiService
+    {
+        private static HttpClient Client;
+        private static String accessToken = "";
+
+        public CHESApiService(
+            ApiDbContext context,
+            IHttpContextAccessor httpContext)
+            : base(context, httpContext)
+        { }
+
+        private static async Task<HttpClient> InitHttpClientAsync()
+        {
+            // if (PrimeConstants.ENVIRONMENT_NAME == "local")
+            // {
+            //     return null;
+            // }
+
+            HttpClient client = new HttpClient();
+
+            if (accessToken == "")
```

Comment:
JWT tokens expire, often in about 5 minutes. This should probably fetch a new token every time, or manage token expiry.